jblosc2/src/main/java/com/jblosc2/BloscWrapper.java

```diff
@@ -181,33 +184,39 @@ public int compress(int compressionLevel, int shuffleType, int typeSize, ByteBuf
 		dest.position(0);
 		src.order(ByteOrder.nativeOrder());
 		dest.order(ByteOrder.nativeOrder());
-		int w = BloscLibrary.blosc_compress(compressionLevel, shuffleType, new NativeLong(typeSize),
+		int cBytes = BloscLibrary.blosc_compress(compressionLevel, shuffleType, new NativeLong(typeSize),
 				new NativeLong(srcLength), src, dest, new NativeLong(destLength));
-		checkExit(w);
-		return w;
+		checkExit(cBytes);
+		return cBytes;
 	}
 
 	public int decompress(Buffer src, Buffer dest, long destSize) {
 		src.position(0);
 		dest.position(0);
-		return BloscLibrary.blosc_decompress(src, dest, new NativeLong(destSize));
+		int nBytes = BloscLibrary.blosc_decompress(src, dest, new NativeLong(destSize));
+		checkExit(nBytes);
+		return nBytes;
 	}
 
 	public int compressCtx(ContextCparams context, Buffer src, long srcLength, Buffer dest, long destLength) {
```

Comment:
Hmm, we are using the `context` name for both `ContextCparams` and `ContextDParams` types.  I suggest to use `CParams` and `DParams` instead.